.gitignore

```diff
@@ -22,3 +22,5 @@ bin
 *.swp
 *.swo
 *~
+
+ocs-osd-deployer
```

Comment:
Is this change by mistake?

Reply:
no, ocs-osd-deployer is the executable build by calling "go build", but we'd never commit the binary so I wanted to gitignore it.

Reply:
Ack, was thinking that this was something else.